features/frameworks/nanostack-libservice/mbed-client-libservice/ns_types.h

```diff
@@ -149,11 +144,7 @@ typedef int_fast32_t int_fast24_t;
  *    static int foo(void) MAYBE_UNUSED;
  * ~~~
  */
-#if defined __CC_ARM || defined __GNUC__
```

Comment:
This should be
```
#if defined __GNUC__
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/ec33c1d353fb7ad7b6300ec060f781f22f54b50d..209f1e6a7190323202b9f9049230307ff8e9a387).